File: src/com/cs/voyage/Poste.cs
public class Poste {
 {
    private int id;
    private string nom;
    private object taux_horaire;

    public Poste(int id, string nom, object taux_horaire) {
        this.id = id;
        this.nom = nom;
        this.taux_horaire = taux_horaire;
    }
    public int Id {
        get { return id; }
        set { id = value; }
    }
    public string Nom {
        get { return nom; }
        set { nom = value; }
    }
    public object Taux_horaire {
        get { return taux_horaire; }
        set { taux_horaire = value; }
    }
}
    }
}

File: src/com/classe/voyage/Poste.java
package com.classe.voyage;

import java.lang.Integer;
import java.lang.Object;
import java.lang.String;

public class Poste {
  private Integer id;

  private String nom;

  private Object taux_horaire;

  public Poste(Integer id, String nom, Object taux_horaire) {
    this.id = id;
    this.nom = nom;
    this.taux_horaire = taux_horaire;
  }

  public Integer getId() {
    return id;
  }

  public void setId(Integer id) {
    this.id = id;
  }

  public String getNom() {
    return nom;
  }

  public void setNom(String nom) {
    this.nom = nom;
  }

  public Object getTaux_horaire() {
    return taux_horaire;
  }

  public void setTaux_horaire(Object taux_horaire) {
    this.taux_horaire = taux_horaire;
  }
}
